### ISSUE TITLE
id-match rule does not work with opaque types

### ISSUE THREAD

**Issue by gajus** [gajus/eslint-plugin-flowtype#377]
Configuration:

```json
"flowtype/type-id-match": [
  2,
  "^([A-Z][a-z0-9]+)+Type$"
],
```

Expected behaviour:

```js
// raise warning
type GoogleApiKey = string;

// raise warning
export opaque type DarkskyApiKey = string;

```

Actual behaviour:

```js
// raises warning
type GoogleApiKey = string;

// does not raise warning
export opaque type DarkskyApiKey = string;

```
